scripts/windows/orange-conda.nsi

```diff
@@ -561,7 +561,7 @@ Section -Launchers
     # to the console that remains visible after exit
     CreateShortCut \
         "$InstDir\${LAUNCHER_SHORTCUT_NAME} Debug.lnk" \
-        "%COMSPEC%" '/K "$PythonExecPrefix\python.exe" -m "{LAUNCHERMODULE} -l4' \
+        "%COMSPEC%" '/K "$PythonExecPrefix\python.exe" "-m ${LAUNCHERMODULE}" -l4' \
```

Comment:
Note that this whole `/K "$PythonExecPrefix\python.exe" "-m ${LAUNCHERMODULE}" -l4` will be recorded verbatim as the shortcut parameters. Therefore the quotes must not group both `-m` and `${LAUNCHERMODULE}` into a single argument. In  fact neither require it as they are by definition single tokens (they cannot contain white space).

Reply:
I see, the quotes are around the third parameter to the CreateShortCut. Fixed.